src/store/concepts/session/operations/logout.js

```diff
@@ -0,0 +1,15 @@
+import { createLogic } from 'redux-logic'
+import Cookies from 'js-cookie'
+
+import { USER_LOGOUT } from 'Store/concepts/session/types'
+
+const logoutUserLogicOperation = createLogic({
+  type: USER_LOGOUT,
+  async process(_, dispatch, done) {
+    Cookies.remove('session_id')
```

Comment:
Is removing cookie enough to logout user? As I understand you also should redirect user to login screen and you can do it only with changing state or direct redirect to page

Reply:
Not enough, I add request on api and make full logout